scripts/process/histWuClean.R

```diff
@@ -6,8 +6,23 @@ process.histWuClean <- function(viz){
   wuAwuds <- readData(viz[['depends']][['histWaterUse']])
   wuData <- readData(viz[['depends']][['waterUse']])
   wuData2015 <- readData(viz[['depends']][['water15']])
-  
+  waterMT <- readData(viz[['depends']][['waterMT']])
   wuData$state_cd <- zeroPad(wuData$state_cd, 2)
+  wuData <- bind_rows(wuData, waterMT)
+  
+  mt_2010 <- filter(wuData, 
+                    state_cd == "30",
+                    year == 2010) %>%
+    filter(!is.na(value))
+  
+  mt_2010_tots <- mt_2010[1,]
+  mt_2010_tots$category[1] <- "Total"
+  mt_2010_tots$value[1] <- sum(mt_2010$value)
```

Comment:
could these MT specific things happen in the MT clean script? that way we can easily change back to pulling from NWIS in the future if we need to?

Reply:
There's some MT data in the wuData that we need, so no...it's needs to be done here.